test/performance/README.md

```diff
@@ -1,128 +1,168 @@
+
+   - 1 deployments, with two pod replicas (pause), mounting 2 secrets, 2 config maps
+   - 3 services, the first service points to the TCP/8080 port of the deployments, respectively.
+   - 10 secrets containing a 2048-character random string.
+   - 10 config maps
+
+   For Openshit clusters: 
+
+   ```bash
+   $ ./setup-clusterloader2.sh --env ocp
+   ```
+
+    In Openshift environments, a modified version of the kube-burner [cluster-density-ms](https://github.com/kube-burner/kube-burner-ocp/tree/478bb42e1842a94ca3210d26a08633b70a443005/cmd/config/cluster-density-ms) workload will be generated per namespace. This workload consists of the following objects:
+
+    - 1 image stream.
+    - 4 deployments, each with two pod replicas (pause), mounting 4 secrets, 4 config maps, and
+      1 downward API volume each.
+    - 2 services, each pointing to the TCP/8080 and TCP/8443 ports of the first and second
+      deployments, respectively.
+    - 1 edge route pointing to the first service.
+    - 20 secrets containing a 2048-character random string.
+    - 10 config maps containing a 2048-character random string
+
+
+3. Configure the parameters of your experiment:  
 
    a) cd into the load configuration directory
 
-```bash
-$ cd  perf-tests/clusterloader2/testing/load
-```
-   b) create the `stress-test` namespace - you workload needs to be deployed into this exact namespace (temporary solution)
-
-```bash
-$ kubectl apply -f - <<EOF
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: stress-test
-  labels:
-    app.kubernetes.io/cluster: wec1
-EOF
-```
-
-   c) configure your sample workload: 
-   
-```bash
-$ vi test-workload.yaml
-```
+   ```bash
+   $ cd  $WORK_DIR
+   ```
   
-   d) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
    
-```bash
-$ vi performance-test-config.yaml
-``` 
+   ```bash
+   $ vi performance-test-config.yaml
+   ``` 
+
+   More specifically, configure the following parameters: 
+
+   - namespaces: number of namespaced (*default value: 1*)
+   - namespaceNamePrefix: name prefix for the namespace (*default value: "perf-exp"*)
+   - K8S_CLUSTER: set to `true` for Kubernetes clusters (*default value: "true"*)
+   - OPENSHIFT_CLUSTER: set to `true` for Kubernetes clusters (*default value: "false"*)
+   - tuningSet: workload generation function (*default value: "RandomizedLoad"*)
+
+   To learn more about the clusterloader2 load experiments configuration parameters, see the following: [clusterloader tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set).
```

Comment:
That link only leads to documentation of `TuningSet`. There is much more than TuningSet involved here. The reader needs to know where to read to understand all of the paramters she was just asked to edit.

BTW, the TuningSet documentation is way out of date. It lists only 3 cases. There are now 8.

Reply:
Yes, the clusterloader documentation from the community repo is out of date indeed. I added a new link that shows where to learn more about the parameters to configure tunningSet

Reply:
My main point here is that the reader needs to know about a lot more than TuningSet. The reader wonders, what is the meaning of all those parameters I was just told to edit? How do I decide what values to set those parameters to? (the two Booleans are obvious and you already have a link for the TuningSet; my question here is the first two listed parameters.)

Reply:
Ok, I see! You're referring to these two parameters: `namespaces` and `namespaceNamePrefix`. I just add more description to explain the meaning of those parameters